src/common/helpers/axiosClient.ts

```diff
@@ -1,17 +1,15 @@
 // #region Global Imports
+import {LANG_TYPES} from "@constants/enums";
 import axios, {AxiosInstance, AxiosRequestConfig, AxiosResponse} from "axios";
 import {Buffer} from "buffer";
 
-import {LANG_TYPES} from "../constants/enums";
 import storage from "./localStorage";
+
 const configs: AxiosRequestConfig = {
-  baseURL: "http://localhost:3000/api",
-  // baseURL: `${process.env.SERVER_API_URL}`,
+  baseURL: process.env.SERVER_API_URL || "/api",
   headers: {
     "Content-Type": "application/json",
     "Cache-Control": "no-cache",
-    "Access-Control-Allow-Origin": "*",
-    "Accept-Language": storage.getLanguage() || LANG_TYPES.ENGLISH,
```

Comment:
I think set `Accept-Language` is redundant as we will set this header again at [request interceptor](https://github.com/TyronNhatAnh/base-nextjs/pull/1/files#diff-6c1428453765c4a0d524038d33cd2078993873dfd3f1d7b2af272150824ebf86R25) anyway